### ISSUE TITLE
I work on Jaeger UI; we'd love to hear any feedback or suggestions

### ISSUE THREAD

**Issue by tiffon** [aliyun/aliyun-log-jaeger#11]
Hello,

TLDR; I work on [Jaeger UI](https://github.com/jaegertracing/jaeger-ui), we'd love to hear any feedback or suggestions, or even to just chat about tracing!

I'm a maintainer on the Jaeger project, focused mostly on Jaeger UI.

I wanted to reach out to say we're thrilled that you've embraced Jaeger!

We're constantly striving to improve Jaeger, and we see the extensibility of the platform and UI as central to that.

We're very keen to learn what your experience with Jaeger has been like.

* What pain-points did you encounter?
* What's working well?
* Any suggestions?

Any feedback would be fantastic :+1:

We can also share our plans for new features in development, such as path-based dependency graphs, to gather your feedback while they're in development.

Please let me know what you think.

Best.

**Comment by tiffon** [comment]
@brucewu-fly Just wanted to check in to see if you're interested in chatting about Jaeger?

**Comment by brucewu-fly** [comment]
@tiffon I'm very interested in how to implement path-based dependency graphs. Can you share your idea or some documents?
And in the future, I hope the aliyun-log plugin can merge back to the community.
